README.md

```diff
@@ -285,4 +285,5 @@ git commit -a --amend --no-edit
 bundle exec rake release
 ```
 
-[github action](.github/workflows/ruby-gem-publication.yml) will release a new version to rubygems.org
+- [github action](.github/workflows/ruby-gem-publication.yml) will release a new version to rubygems.org
+- you need to manually open the action for the new version commit in github and click the approve button for the action
```

Comment:
Perhaps add a link to https://github.com/zendesk/stronger_parameters/actions/workflows/ruby-gem-publication.yml

Reply:
added